app/static/js/app.js

```diff
@@ -317,6 +323,32 @@ document.getElementById("fullscreen-btn").addEventListener("click", (evt) => {
 document.getElementById("paste-btn").addEventListener("click", () => {
   showPasteOverlay();
 });
+
+document.getElementById("update-btn").addEventListener("click", async () => {
+  const { version } = await getVersion();
+  const { latestRelease } = await getLatestRelease();
+  const updateDialog = document.getElementById("update-dialog");
+
+  if (version === latestRelease) {
+    updateDialog.update = false;
+  }
+  updateDialog.update = true;
```

Comment:
Damn, idk why I put this below the `if` before sending to you, totally missed on my last minute review. Sorry. @mtlynch